packages/web-react/config/jest/config.js

```diff
@@ -25,6 +25,9 @@ const config = {
     '^.+\\.(t|j)sx?$': ['<rootDir>/../../node_modules/@swc/jest'],
   },
 
+  // https://jestjs.io/docs/configuration#transformignorepatterns-arraystring
```

Comment:
Please, also add a short description as it is above every other option. Eg. those description is from that link. Most of the time it is the first sentence from docs that describes perfectly what the option does. Sometimes it needs a little bit of modification.